modules/apps/export-import/export-import-service/src/main/java/com/liferay/exportimport/internal/lar/PortletDataContextImpl.java

```diff
@@ -948,15 +948,6 @@ public Object getNewPrimaryKey(String className, Object newPrimaryKey) {
 		return _newPrimaryKeysMaps;
 	}
 
-	/**
```

Comment:
Hi @dantewang @yuhai ,
This actually changed the info we dump into file. Even though we are sure on one is using it, it is still risky. 
Considering we are going to replace the whole staging with change tracking soon, I would like to hold on for this one and play safe for this round of deprecation removal.

Thanks.
Tina.